packages/@okta/vuepress-theme-prose/assets/css/components/_search.scss

```diff
@@ -1,5 +1,5 @@
 .proxima {
-  font-family: "proxima-nova", sans-serif;
+  font-family: 'ABC Whyte', "proxima-nova", sans-serif;
```

Comment:
Same comment as in the typography section, we should not fall-back to proxima. We can remove it from the list if appropriate. We may not even actually need this line any more if the font settings cascade.

Reply:
It overwrites library styles, it has Lato font hardcoded